rockspec/apisix-2.10.3-0.rockspec

```diff
@@ -0,0 +1,95 @@
+--
```

Comment:
Maybe we don't need to archive this rockspec? We don't merge the code into the default branch.

Reply:
@spacewander Here it is marked for installation using the `2.10.3` git tag, as far as I know this could potentially be used when packaging various container images?

[https://github.com/apache/apisix-docker/blob/5d3c4c63d21f0c40655d005fdb9cb3746f2e4af3/alpine/Dockerfile#L22](https://github.com/apache/apisix-docker/blob/5d3c4c63d21f0c40655d005fdb9cb3746f2e4af3/alpine/Dockerfile#L22)

Reply:
Hi, @spacewander. Do I need to make changes for this?

Reply:
I see.